tools/docker/builder/openwrt/build.sh

```diff
@@ -177,7 +177,7 @@ main() {
 IMAGE_ONLY=false
 OPENWRT_REPOSITORY='https://git.prpl.dev/prplmesh/prplwrt.git'
 OPENWRT_VERSION='30c0f8b1e23a59c3e15c4eb329d5689b55280529'
-PRPL_FEED='https://git.prpl.dev/prplmesh/feed-prpl.git^fcae5377c6e0d8c64073e47033f131a8e7d1b965'
+PRPL_FEED='https://git.prpl.dev/prplmesh/feed-prpl.git^b230cc95f42d7093c04960a01d254d497f6e7409'
```

Comment:
This obviously has to be updated to the actual commit hash after merging !2. So I'm going to add the don't merge tag.

Reply:
Actually we're using FF merge so the commit SHA1 wasn't changed.